marlowe-runtime-web/server/Language/Marlowe/Runtime/Web/Server/TxClient.hs

```diff
@@ -128,6 +143,7 @@ txClient :: forall r. Component IO (TxClientDependencies r) (TxClient r)
 txClient = component \TxClientDependencies{..} -> do
   tempContracts <- newTVar mempty
   tempTransactions <- newTVar mempty
+  tempWithdrawals <- newTVar mempty
```

Comment:
So contracts, transactions, and withdrawals are the ephermeral local state of the webserver? Is there some sort of garbage collection on these, or will the maps get larger and larger as clients choose not to sign and submit some transactions? I suppose that unsigned transactions could be garbage collected when their validity interval passes.